test/store-listener-component-test.js

```diff
@@ -203,6 +255,18 @@ export default {
       assert(span.props.TestStore.x === 'foobar')
     },
 
+    'children props work with the transform function'() {
+      const node = TestUtils.renderIntoDocument(
+        <AltContainer stores={{ TestStore }} transform={transformMock}>
```

Comment:
I want to test the transform function has been called and get the same TestStore as I pass to it.

But I don't want to test it for every possibilities about props because all the other tests about props are passed which means the default transform function is ok.

What should I do if I want to make it more clear?